src/munge.spec.ts

```diff
@@ -31,6 +37,29 @@ const validateOfferCodecs = (offer: Sdp): boolean => {
   return true;
 };
 
+/**
+ * Check that the sdp offer contains rtcpFbValue or not.
+ *
+ * @param offer - The {@link Sdp} or {@link AvMediaDescription} to validate.
+ * @param rtcpFbValue - The rtcp-fb value to check.
+ * @returns True if the offer contains rtcp-fb value.
+ */
+const checkOfferContainsRtcpFeedbacks = (
```

Comment:
Nit: plural of "feedback" is still "feedback". This function should be called `checkOfferContainsRtcpFeedback`.

Reply:
udpated.